apps/arweave/src/ar_mining_hash.erl

```diff
@@ -0,0 +1,128 @@
+-module(ar_mining_hash).
+
+-behaviour(gen_server).
+
+-export([start_link/0, compute_h0/2, compute_h1/2, compute_h2/2]).
+
+-export([init/1, handle_cast/2, handle_call/3, handle_info/2, terminate/2]).
+
+-include_lib("arweave/include/ar.hrl").
+-include_lib("arweave/include/ar_config.hrl").
+-include_lib("arweave/include/ar_consensus.hrl").
+-include_lib("arweave/include/ar_mining.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+-record(state, {
+	hashing_threads				= queue:new(),
```

Comment:
```suggestion
	hashing_threads = queue:new(),
```

Reply:
this formatting is consistent with the ar_mining_server formatting (which is where this file was split out from). I don't feel strongly about whether it's better to line up the assignments or not (or whether it's even important enough to standardize), but I'm reluctant to change it in this one place since it is consistent with how this group of files was formatted in the past.

Reply:
they didn't line up the equal sign on line 16 and 17, could be github's graphical representation, feel free to ignore

Reply:
yeah, probably the tabs vs. spaces issue.

Reply:
If we want ot move the needle on that, my suggestion is we break it into two steps:
1. enable the linter and have it enforce the current repo standards (e.g. tabs)
2. once that's enabled and running smoothly (and if we decide that spaces really is better than tabs), we find a time to convert the whole repo to spaces and update the linter rules

Step 1 I think could be very useful. I don't feel strongly about step 2 so long as we can find a time to do the conversion.